### PR TITLE
tracker component, and routes with mock-up

### PR DESCRIPTION
**Pull request by carlos-ch**

### CODE REVIEW

File: src/trackerConfig.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    { name: 'breakfast', text: 'I had breakfast' },
+    { name: 'lunch', text: 'I had lunch' },
+    { name: 'tea', text: 'I had tea' },
+    { name: 'dinner', text: 'I had dinner' },
+  ],
+};
+
+const water = {
+  title: 'Water tracker',
+
```
Comment (emirodriguezalc) [inline src/trackerConfig.js:14]: Space?

File: src/views/tracker/styles.css [hunk 2]
```diff
@@ -0,0 +1,77 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+body {
+  display: flex;
+  font-family: 'Roboto', sans-serif;
```
Comment (emirodriguezalc) [inline src/views/tracker/styles.css:7]: we have to remember to implement this on app body, not here. ill do it in the next pr

File: src/views/tracker/styles.css [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  text-align: center;
+}
+.food-menu {
+  width: 100vw;
+  height: 100vh;
+  background-color: #f2d7bf;
+}
+.water-menu {
```
Comment (emirodriguezalc) [inline src/views/tracker/styles.css:15]: lets try to be consistent on the spacing. or we add it always or we don't. I prefer spaces but we can talk about it

File: src/views/tracker/toggle.css [hunk 4]
```diff
@@ -0,0 +1,22 @@
+input[type='checkbox'] {
+  display: none;
+}
+
+input[type='checkbox'] + label {
+  background-image: url(../../icons/toggle-no.png);
```
Comment (emirodriguezalc) [inline src/views/tracker/toggle.css:6]: no quotes?

File: src/views/tracker/toggle.css [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  background-size: cover;
+  background-position: center;
+  height: 50px;
+  width: 180px;
+  display: inline-block;
+  padding: 0;
+}
+
+input[type='checkbox']:checked + label {
+  background-image: url(../../icons/toggle-yes.png);
```
Comment (emirodriguezalc) [inline src/views/tracker/toggle.css:16]: no quotes?

File: src/views/tracker/toggle.css (hunk 4 above)
Comment (carlos-ch) [reply src/views/tracker/toggle.css:6]: nope

File: src/views/tracker/toggle.css (hunk 5 above)
Comment (carlos-ch) [reply src/views/tracker/toggle.css:16]: nope

File: src/trackerConfig.js (hunk 1 above)
Comment (carlos-ch) [reply src/trackerConfig.js:14]: I see

**Pull request merged**